### DISCUSSION TITLE
Daily Dev Bookmarks to Notion

### DISCUSSION THREAD

**Discussion by dhruvwill** [dailydotdev/daily#933] [Feature Requests]
Feature: Daily Dev Bookmarks are automatically added to Notion Page.

**Comment by nimrodkra** [comment]
Check out the Share Bookmarks feature. Using this option you can create an integration that would automatically add bookmarks to your Notion: https://docs.daily.dev/docs/integrations/sharing-bookmarks

Let us know if you ended up using it! Good luck
